modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -2375,6 +2697,12 @@ By convention, some values are forbidden. It must be greater than 1024 and diffe
 | `object`
 | scheduling controls how the pods are scheduled on nodes.
 
+| `secondaryNetworks`
+| `array`
+| Define secondary networks to be checked for resources identification.
+In order to guarantee a correct identification, it is important that the indexed values form an unique identifier across the cluster.
+If there are collisions in the indexes (same index used by several resources), those resources might be wrongly labeled.
```

Comment:
```suggestion
If the same index is used by several resources, those resources might be wrongly labeled.
```

Reply:
I also think incorrectly is better than wrongly. Thanks for pointing this out.